api/src/db/mod.rs

```diff
@@ -0,0 +1,89 @@
+//! Database business layer
+//!
+//! Contains actions to interact with a database using `Diesel`
+
+#[macro_use]
+pub mod models;
+mod schema;
+
+use self::models::{NewUser, User};
+use diesel::delete;
+use diesel::insert_into;
+use diesel::prelude::*;
+use rand::{self, Rng};
+use std::env;
+
+/// Returns a database connection, used for debug
+#[allow(dead_code)]
+pub fn establish_connection() -> MysqlConnection {
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
+
+    MysqlConnection::establish(&database_url)
+        .expect(&format!("Error connecting to {}", database_url))
```

Comment:
Error format is `error: <details>`, preferably starting with a verb.

Eg: `error: can't connect to {}`

Reply:
useless function removed